en/docs/get-started/tutorials/create-your-first-integration.md

```diff
@@ -0,0 +1,134 @@
+# Create Your First Integration
+
+An integration is a component with logic to integrate APIs, microservices, applications, and data in different languages and formats.
+
+Consider a scenario where a developer has created an integration in [WSO2 Integration Studio](https://wso2.com/integration/integration-studio/) and wants to use it for API-led integration by exposing it in Choreo. In this tutorial, you will learn how to do the following actions to address this requirement:
+
+
+- Expose an integration created in [WSO2 Integration Studio](https://wso2.com/integration/integration-studio/) as a Choreo component by connecting the GitHub repository in which it resides to Choreo.
+- Deploy the Integration component to the development environment.
+- Test the Integration component.
+- Observe statistics for the Integration component.
+- Publish the Integration component and try it out in the production environment.
+
+For this tutorial, let's use a basic sample application that returns `Hello` as the response when you invoke it.
```

Comment:
This sample is not an integration right? I don't think this gives the idea of an integration.

Reply:
This is a simple project to start integration. later they can include different integrations/connectors and extend the sample. This sample demonstrates choreo can deploy MI projects. And also any user can deploy this sample and test it without any code changes. Because if we add actual integration user will not be able to invoke/use without code/config changes

Reply:
Removed the part that describes what the proxy does.